modules/extensions/apps/portal/store/carbon.super/fs/gadget/avg_execution_time_vs_task_id/index.xml

```diff
@@ -60,11 +62,14 @@
 </head>
 
 <body>
-    <div class="panel panel-default">
-        <div class="panel-body">
+    <div>
+        <div class="panel-body" style="width:100%; height:95%">
```

Comment:
Fixed in https://github.com/wso2/product-pc/commit/e931bb704e9065cf57f08f61619a45a4beecb5c7